java/src/main/java/com/logicalclocks/hsfs/Storage.java

```diff
@@ -18,6 +18,5 @@
 
 public enum Storage {
   OFFLINE,
-  ONLINE,
-  ALL
```

Comment:
why removing `ALL` - assuming you want to insert on both online and offline do you have to call the method twice? Do we want that?

Reply:
as we discussed, the logic is now the following:

- we get rid of the default storage, but we keep the storage option in the read and insert methods
- save: saves offline, both if online is enabled
- read: defaults to reading from offline, unless you pass storage online
- insert: same as save, but you can overwrite by passing the storage
- user wants to write only to the online feature store: create empty FG with online enabled, use insert.

so if you don't specify Storage, it will insert in both if online_enabled. if you specify Storage, either Online or offline, it will only insert in one of the two.